config.go

```diff
@@ -287,7 +288,8 @@ func (s String) Type() Type { return StringType }
 
 func (s String) String() string {
 	str := strings.Trim(string(s), `"`)
-	if strings.Contains(string(s), ":") {
+	compile := regexp.MustCompile("[\x20-\x40|\x5b-\x60|\x7b-\x7e]+")
```

Comment:
In the ASCII table, `\x20-\x40|\x5b-\x60|\x7b-\x7e]+` includes `$` / `{` / `}`, I'm not sure if this is correct, please let me know if I'm wrong